src/features/student/views/StudentTraceView/components/UpdateTraceModal/UpdateStep.vue

```diff
@@ -24,7 +24,7 @@ const activeTab = ref(0)
       </AvTab>
       <AvTab
         :title="t('student.views.studentTraceView.updateTraceModal.steps.update.tabs.associations.title')"
-        icon="mdi:chat-alert"
+        :icon="MDI_ICONS.LINK"
```

Comment:
```suggestion
        :icon="MDI_ICONS.CHAT_ALERT"
```
C'est celle que tu viens d'ajouter, c'est pas plutôt ça qu'il faudrait faire ?

Reply:
@p-beger j'ai trouvé qu'on persiste pas l'icon pour une fontionnalité à chaque fois on change
donc pout tous ce qui est associations on garde MDI_ICONS.LINK

<img width="776" height="620" alt="image" src="https://github.com/user-attachments/assets/e649b696-674f-4b32-a086-1305797bec80" />


<img width="1015" height="805" alt="image" src="https://github.com/user-attachments/assets/d5a3be2d-23c4-4db4-99d7-97024bf17492" />